build_r.R

```diff
@@ -56,7 +56,7 @@ result <- file.copy(from = "CMakeLists.txt",
                     overwrite = TRUE)
 .handle_result(result)
 
-# Build the package
+# Build the package (do not touch this line!)
```

Comment:
@jameslamb Not any specific...
It's just the guard from  accidental removal or modification of this line in the future, because it's used as a marker poineted to line from which I cut `build_r.R` script.  Cutting is needed to avoid unnecessary compilation of LightGBM, because I use `build_r.R` only for preprocessing (copying needed files) and compilation occures later during the call of `devtools::document()`.

> sed -i'.bak' '/# Build the package (do not touch this line!)/q' build_r.R

This `sed` command drops all lines of the file after the specified one.